### ISSUE TITLE
area-alt rule reports wrong error message (copy-paste from aria-allowed-attr)

### ISSUE THREAD

**Issue by koddsson** [koddsson/accessibility-scanner#289]
## Bug

In `src/rules/area-alt.ts`, the error message says:

> "Elements must only use allowed ARIA attributes"

This is the description for the `aria-allowed-attr` rule, not `area-alt`. The correct message should be something like:

> "Ensures \<area\> elements of image maps have alternate text"

Likely a copy-paste error from when the rule was created.
